### ISSUE TITLE
cert.pem key to small, so build on stock Debian fails cunit tests

### ISSUE THREAD

**Issue by rjbs** [cyrusimap/cyrus-imapd#2961]
```
Server accepted connection
S: * ccSTARTTLS
S: * FOO BAR
S: * BAZ
S: * QUUX FOONLY FMEH
S: * FNORD BOO GNERGH
S: * cApItAls Oslo
S: OK Toy Test server v0.0.1
C: XXSTARTTLS


SYSLOG 7[TLS server engine: No client CA data configured.]

SYSLOG 3[unable to get certificate from '/home/rjbs/code/cyrus-imapd/cunit/cert.pem': ee key too small]

SYSLOG 3[TLS server engine: cannot load cert/key data, may be a cert/key mismatch?]
S: BAD error initializing TLS

SYSLOG 3[couldn't authenticate to backend server: no mechanism available]
C: XXLOGOUT
```

**Comment by elliefm** [comment]
Probably fixed by #2870

**Comment by elliefm** [comment]
Also previously discussed in #2470

**Comment by elliefm** [comment]
Should be fixed by #3033
